sigs.yaml

```diff
@@ -2027,6 +2027,10 @@ sigs:
     - name: sig-storage-test-failures
       description: Test Failures and Triage
   subprojects:
+  - name: cosi
```

Comment:
@saad-ali Do we want this subproject to be called kubernetes-cosi or just cosi?
We have subproject named kubernetes-csi on line 2046.  Should this one be consistent and be called kubernetes-cosi?

Reply:
Yeah, I would prefer kubernetes-cosi as well

Reply:
Updated to `kubernetes-cosi`